templates/widget/__tests__/widget.test.ts

```diff
@@ -0,0 +1,33 @@
+import 'reflect-metadata';
```

Comment:
The name of the file should likely align with the name of the actual widget.

Reply:
I am not sure what this means? The file name is parameterized.

Reply:
I see now that the file path is parameterized, just that this is the template path :+1: 
https://github.com/eclipse-theia/generator-theia-extension/pull/112/files#diff-e234283aa9cd1cdf0803a13194964671ae1dfd31064f5fe35429df36a51ef17bR334